src/index.html

```diff
@@ -7,8 +7,71 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image__area">
+        <a href="#Home">
+          <img
+          class="icon"
+          src="images/logo.png"
+          alt="image"
+          >
+        </a>
+      </div>
+
+      <nav class="nav__list">
+        <ul class="list">
+          <li class="list__item">
+            <a href="#Apple" class="link is-active">
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#Apple" class="nav__link is-active">
```

Reply:
If you strive to give proper class names for elements, consider doing it this way